### ISSUE TITLE
Doesn't build with current dmd or ldc due to stream.d usage

### ISSUE THREAD

**Issue by antonio-rojas** [ximion/appstream-generator#42]
stream.d has been removed from phobos, so asgen can't be built with the latest releases of dmd (2.074) or ldc (1.2.0)

FAILED: appstream-generator@exe/src_asgen_result.d.o 
ldc2  '-Iappstream-generator@exe' '-I.' '-I../appstream-generator-0.6.3' '-I../appstream-generator-0.6.3/src/' '-I../appstream-generator-0.6.3/src/girepo/' '-I/usr/include/d/mustache-d' '-I/usr/include/glib-2.0' '-I/usr/lib/glib-2.0/include' '-I/usr/include/AppStream' '-I/usr/include/cairo' '-I/usr/include/pixman-1' '-I/usr/include/freetype2' '-I/usr/include/harfbuzz' '-I/usr/include/libdrm' '-I/usr/include/libpng16' '-I/usr/include/gdk-pixbuf-2.0' '-I/usr/include/librsvg-2.0' '-I/usr/include/pango-1.0' '-enable-color' '-wi' '-g' '-O0' '-enable-cross-module-inlining=false'  -of 'appstream-generator@exe/src_asgen_result.d.o' -c ../appstream-generator-0.6.3/src/asgen/result.d
../appstream-generator-0.6.3/src/asgen/result.d(207): Error: module stream is in file 'std/stream.d' which cannot be read

**Comment by ximion** [comment]
Hah!
This just just a stray import without any meaning, asgen never used `std.stream` as it has been deprecated for years.

tl;dr You can just remove that line, and I'll submit a patch removing it when home.
